app/templates/macros/answers.html

```diff
@@ -0,0 +1,66 @@
+{% macro textbox(value='') -%}
```

Comment:
Will these macros eventually be shared between apps - in either the utils package or frontend toolkit?